### DISCUSSION TITLE
Overscroll-behavior as default or configuration

### DISCUSSION THREAD

**Discussion by jonatassalgado** [buzinas/simple-scrollbar#68] [Q&A]
Make sense to set `overscroll-behavior` to `contain` as default or add in configuration in lib? 

In my use case like a chat container or modal with scroll, the scroll of this elements affect the window scroll, causing a unexpected behaviour, like this example from google developers: https://developers.google.com/web/updates/2017/11/overscroll-behavior 

Othre suggestion to apply the `overscroll-behavior` without a pull request? 
The attribute need be set in `ss-content` element.

**Comment by buzinas** [comment]
Well, theoretically you just need to add `.ss-content { overscroll-behavior: contain; }` to your stylesheet.
